src/client/themes/classic/index.ts

```diff
@@ -63,18 +105,17 @@ const classicTheme: DefaultTheme = {
       },
     },
 
-    walletButton: {
-      background: '#E5E5E5',
-      color: '#006AE3',
-    },
+    // walletButton: {
+    //   background: '#E5E5E5',
+    //   color: '#006AE3',
+    // },
```

Comment:
Remove ?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6